### PR TITLE
@sweir27 => [Artist] Add artist series to works tab

### PR DESCRIPTION
**Pull request by mzikherman**
Looks like

search

search

Pretty simple, reused the same component as in https://github.com/artsy/force/pull/6000

Will add some tests.

### CODE REVIEW

File: src/v2/Apps/Artist/Routes/Works/index.tsx [hunk 1]
```diff
@@ -21,14 +24,21 @@ export const WorksRoute: React.FC<WorksRouteProps> = props => {
...
   return (
     <>
       <Box>
         <ArtistTopWorksRail artist={artist} />
       </Box>
 
       <Box>
-        <ArtistCollectionsRail artistID={artist.internalID} />
+        {artistSeriesIsEnabled ? (
+          <ArtistSeriesRail artist={artist} />
```
Comment (mzikherman) [inline src/v2/Apps/Artist/Routes/Works/index.tsx:38]: This takes the place of the collections rail (which is lazily loaded, and now we get full SSR for the series).

File: src/v2/Apps/Artist/Routes/Works/index.tsx [hunk 2]
```diff
@@ -77,9 +87,11 @@ export const WorksRouteFragmentContainer = createFragmentContainer(WorksRoute, {
         sizes: { type: "[ArtworkSizes]" }
         sort: { type: "String", defaultValue: "-partner_updated_at" }
         width: { type: "String" }
+        shouldFetchArtistSeriesData: { type: "Boolean!", defaultValue: false }
       ) {
       internalID
       ...ArtistTopWorksRail_artist
+      ...ArtistSeriesRail_artist @include(if: $shouldFetchArtistSeriesData)
```
Comment (mzikherman) [inline src/v2/Apps/Artist/Routes/Works/index.tsx:94]: If the user doesn't have the lab feature, they don't need to fetch the fragment. Saves a little bit of data requesting and bandwidth. Did the same thing for the artwork page 'other works from this series' rail (https://github.com/artsy/force/blob/4301e22db2ffe9c5da0859c46e967621f65c69b4/src/v2/Apps/Artwork/routes.tsx#L17)

File: src/v2/Apps/Artist/routes.tsx [hunk 3]
```diff
@@ -222,6 +225,12 @@ export const routes: RouteConfig[] = [
             }
           )
 
+          const user = props.context.user
+          filterParams.shouldFetchArtistSeriesData = userHasLabFeature(
```
Comment (mzikherman) [inline src/v2/Apps/Artist/routes.tsx:229]: This is the output of `prepareVariables`, so it should match the shape of the GraphQL variables. For the artwork page (no filter query params), that's in https://github.com/artsy/force/blob/4301e22db2ffe9c5da0859c46e967621f65c69b4/src/v2/Apps/Artwork/routes.tsx#L21

**Comment by mzikherman** [comment]
Updated with specs.

**Comment by sweir27** [comment]
Looks good to meee 🚀

**Pull request merged**